tests/functional/d/disable_msg_next_line.py

```diff
@@ -18,3 +18,10 @@ def function_C():
 
 def function_D(arg1, arg2):  # [unused-argument, invalid-name]
     return arg1
+
+
+def function_E():  # [invalid-name]
+    # pylint: disable-next=unused-variable
+
```

Comment:
```suggestion
bleh = 42
```

Reply:
?

Reply:
(To check that the disable next do work on the next line)

Reply:
Nvm, I did not read the issue properly.

Reply:
😄 

Rest of the file tests that as well!